### ISSUE TITLE
index page with links to all cities

### ISSUE THREAD

**Issue by arciitek** [avernois/devfriendlyplaces#137]
hi there,

is it possible to create an index page or pane that allows people to navigate to the city they are looking for, also, defaulting to the city closest to the visitors location would be good..

Cordialement

**Comment by avernois** [comment]
Yep, a first index page is coming with PR #132 that will probably merged today :)

**Comment by francois2metz** [comment]
Can be closed now ;)
